std/conv.d

```diff
@@ -2218,7 +2221,16 @@ if (isInputRange!Source &&
             static if (isNarrowString!Source)
                 source = cast(Source) s;
 
-            return result;
+            static if (doCount)
+            {
+                if (result)
+                    return tuple(result, 4);
+                return tuple(result, 5);
```

Comment:
Can you please change the code to return a named tuple?
```
return tuple!("data", "count")(result, 5);
```
Thus, the result can be accessed as `result.data` and `result.count`, making the code easier to read.
After looking at the unittests, I think this change would be a nice addition, as it will make it easier to read, rather than using `result[0]` (for data) and `result[1]` (for count). It's also less error prone; it's easy to use the wrong index, but the named version will be impossible to mistake.